website/cue/reference/releases/0.10.0.cue

```diff
@@ -66,7 +66,7 @@ releases: "0.10.0": {
 		{sha: "e01fc0b69c335efb426aaee30dc9bec442c99ea2", date: "2020-05-01 20:54:20 +0000", description: "Dynamically derive the list of paths to create at target", pr_number:                          2525, scopes: ["operations"], type:                                 "chore", breaking_change:       false, author: "MOZGIII", files_count:                 1, insertions_count:   1, deletions_count:     22},
 		{sha: "8d1f6205e1b262d6111b1d27df0c1f95fe7b6563", date: "2020-05-01 14:12:17 +0000", description: "Run entire healthcheck lazily", pr_number:                                                     2501, scopes: ["kafka sink"], type:                                 "fix", breaking_change:         false, author: "Lucio Franco", files_count:            1, insertions_count:   3, deletions_count:     1},
 		{sha: "2937c46e0f8f061c120a8c21840cc3101434266c", date: "2020-05-01 14:12:35 +0000", description: "Properly detect shutdown", pr_number:                                                          2429, scopes: ["socket sink"], type:                                "fix", breaking_change:         false, author: "Lucio Franco", files_count:            3, insertions_count:   142, deletions_count:   6},
-		{sha: "105ab797c02e0b6787fd56667d020beb80df3e88", date: "2020-05-01 14:37:02 +0000", description: "Align Github Actions workflows with the files changed", pr_number:                             2517, scopes: ["operations"], type:                                 "chore", breaking_change:       false, author: "Binary Logic", files_count:            7, insertions_count:   130, deletions_count:   99},
+		{sha: "105ab797c02e0b6787fd56667d020beb80df3e88", date: "2020-05-01 14:37:02 +0000", description: "Align GitHub Actions workflows with the files changed", pr_number:                             2517, scopes: ["operations"], type:                                 "chore", breaking_change:       false, author: "Binary Logic", files_count:            7, insertions_count:   130, deletions_count:   99},
```

Comment:
It's likely that these shas here relate to commits. I doubt that these are used as test data. If they're just for a website, then there's potentially some value in using correct spelling for humans to read/find. But, if this is intentionally designed to be changelog content and the repository doesn't like changing changelogs, I'm happy to drop this (and following).

Reply:
cc @jszwedko for thoughts - I _think_ these are populated by the commit messages, but I'm not really aware of how we feel about changing changelogs (do you need an item in the changelog for changing the changelog? 😂 )